src/webresource-handler/index.ts

```diff
@@ -193,17 +183,12 @@ export const getUploaderMiddlware = (
 				next();
 			} catch (err: any) {
 				await clearFiles();
-
-				if (err instanceof FileSizeExceededError) {
-					return sbvrUtils.handleHttpErrors(
-						req,
-						res,
-						new errors.BadRequestError(err.message),
-					);
-				}
-
-				getLogger(getApiRoot(req)).error('Error uploading file', err);
-				next(err);
+				getLogger(getApiRoot(req)).warn('Error uploading file', err);
+				return sbvrUtils.handleHttpErrors(
+					req,
+					res,
+					new errors.BadRequestError(err),
```

Comment:
I am slightly conflcited with this one. At one side I belive that the handlers should only use pine as a dev dependency for implementing proper types. On the other hand, not everything is a BadRequestError. I would accept suggestion/ideas here.

Reply:
I think it's perfectly fine for handlers to depend on specific versions of pinejs, expected even, so them throwing specific http errors is fine by me. The only thing I would say is that they should probably use a peer dependency for it